debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/logminer/logwriter/CommitLogWriterFlushStrategy.java

```diff
@@ -33,6 +34,7 @@ public class CommitLogWriterFlushStrategy implements LogWriterFlushStrategy {
     private static final String DELETE_FLUSH_TABLE = "DELETE FROM %s";
 
     private final String flushTableName;
```

Comment:
Does it make sense to rename `flushTableName` to `qualifiedFlushTableName`

Reply:
I'm certainly open to whatever you feel you'd prefer @jpechane.  I left it as since it could theoretically not be qualified if the user provided only the table name and not the schema for backward compatibility reasons.